File: problemas/iniciante/1185/1185.ts
process.stdin.setEncoding('utf-8');

let input: string = '';

process.stdin.on('data', (chunk: string) => {
    input += chunk;
});

process.stdin.on('end', () => {
    let entrada: string[] = input.trim().split('\n');
    let O: string = entrada.shift() as string;

    let soma: number = 0.0;
    let count: number = 0;

    for (let i = 0; i < 12; ++i) {
        for (let j = 0; j < 12; ++j) {
            let valor: number = parseFloat(entrada.shift() as string);

            if (i < 11 - j) {
                soma += valor;
                ++count;
            }
        }
    }

    console.log(`${(O === 'S' ? soma : soma / count).toFixed(1)}`);

});

File: problemas/iniciante/1185/1185.py
def main():
    O = input()
    soma = 0.0
    count = 0

    for i in range(12):
        for j in range(12):
            valor = float(input())

            if i < 11 - j:
                soma += valor
                count += 1
    
    print(f"{soma:.1f}" if O == 'S' else f"{(soma / count):.1f}")

if __name__ == "__main__":
    main()
